### PR TITLE
Pablo Rey precourse challenges inicial commit

### PR DESCRIPTION
**Pull request by paulusrex**

### CODE REVIEW

File: staff/pablo-rey/precourse/battleship/index.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+.ships {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#ships-vertical {
+  display: flex;
+  /* align-items: center; */
```
Comment (edfrias) [inline staff/pablo-rey/precourse/battleship/index.css]: @paulusrex Maybe you don't need this commented code, right?

File: staff/pablo-rey/precourse/pasapalabra-pro/css/main.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+#question-letter-circle {
+  width: 6rem;
+  height: 6rem;
+  margin: 10px auto;
+  border: 3px solid $secondary;
+  border-radius: 50%;
+  text-align: center;
+  transform: translateX(30px)
```
Comment (edfrias) [inline staff/pablo-rey/precourse/pasapalabra-pro/css/main.scss]: @paulusrex even that this code works is better to not forget to end css lines with ;

**Review by edfrias — COMMENTED** [review]
@paulusrex awesome work!

File: staff/pablo-rey/precourse/pasapalabra-pro/css/main.scss (hunk 2 above)
Comment (paulusrex) [reply staff/pablo-rey/precourse/pasapalabra-pro/css/main.scss]: Thanks. The linter usually makes this kind of work, but I don't know why the linter doesn't correcct this lines.

Comment (paulusrex) [reply staff/pablo-rey/precourse/pasapalabra-pro/css/main.scss]: Usually the linter put all semicolons in the code. Thanks for the comment. I will save the entire file lintering. Thanks.

File: staff/pablo-rey/precourse/battleship/index.css (hunk 1 above)
Comment (paulusrex) [reply staff/pablo-rey/precourse/battleship/index.css]: Thanks. It's old code because in first iterations I put letters to show where the ships was

**Pull request merged**